lib/rubygems/server.rb

```diff
@@ -657,7 +657,7 @@ def root(req, res)
       "only_one_executable" => true,
       "full_name" => "rubygems-#{Gem::VERSION}",
       "has_deps" => false,
-      "homepage" => "http://docs.rubygems.org/",
+      "homepage" => "http://guides.rubygems.org/",
```

Comment:
my bad! Thought we hosted it with the rest of the rubygems.org stuff for some reason